### DISCUSSION TITLE
rtl:space-x-reverse doesn't work in <span/> label

### DISCUSSION THREAD

**Discussion by jessemoe** [tailwindlabs/tailwindcss#11383] [Help]
**What version of Tailwind CSS are you using?*
latest

**What build tool (or framework if it abstracts the build tool) are you using?**
postcss-cli 8.3.1, Next.js 18.0.9, webpack 5.28.0

**What version of Node.js are you using?**

v16.0.0

**What browser are you using?**

 Chrome, Safari, 

**What operating system are you using?**

macOS, Windows

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**
rtl:space-x-reverse doesn't work in <span/> label

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead.

**Comment by wongjn** [comment]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) or a remotely-hosted git repository that reproduces the unexpected behavior.
